themes/root/templates/RecordDriver/EDS/export-ris.phtml

```diff
@@ -0,0 +1,108 @@
+<?
+$items = $this->driver->getItems();
```

Comment:
I know that and you are right that platform indepentend export templates are not good. But I thinks there is no other solution for Ebsco as they usually use a foreach loop to print items. thats why EDS was blamed as "not export friendly". But I'll see how I can improve code duplication tomorrow.